File: Assets/Scripts/Losing_Steams.js
var LosingSteamsSpawn : GameObject ;
var steam : GameObject;
static var damaged : boolean = true;
static var bulletsOnMoment : int;

function Update() {
	if(damaged) {
		for(var i = 0; i < bulletsOnMoment; i++) {
			var floorSteam = Instantiate(steam, LosingSteamsSpawn.transform.position, Quaternion.identity);
			floorSteam.rigidbody.AddForce(Vector3(Random.Range(-70.0,100)/70, 1, 0)*20000);
		}
		damaged = false;
		bulletsOnMoment = 0;
	}
}

File: Assets/Scripts/LosingSteams.cs
using UnityEngine;
using System.Collections;

public class LosingSteams : MonoBehaviour {
	
	public GameObject LosingSteamsSpawn;
	public GameObject steam;
	public static bool damaged;
	public static int bulletsOnMoment;

	void Start () {
		damaged = false;
	}
	
	void Update () {
		if(damaged) {
			GameObject floorSteam;
			for(int i = 0; i < bulletsOnMoment; i++) {
				floorSteam = Instantiate(steam, LosingSteamsSpawn.transform.position, Quaternion.identity) as GameObject;
				floorSteam.rigidbody.AddForce(new Vector3(Random.Range(-70f,100f)/70f, 1, 0)*20000);
			}
			damaged = false;
			bulletsOnMoment = 0;
		}
	}
}
